src/pages/events-description/index.js

```diff
@@ -0,0 +1,198 @@
+import EventsStyleDescription from './styled'
+import { useState, useEffect } from 'react'
+import { cms } from '../../service/client'
+import { useParams } from 'react-router'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Swiper, SwiperSlide } from 'swiper/react'
+import { FreeMode, Navigation, Thumbs } from 'swiper/modules'
+import { Link } from 'react-router-dom'
+import env from 'react-dotenv'
+import 'swiper/css'
+import 'swiper/css/free-mode'
+import 'swiper/css/navigation'
+import 'swiper/css/thumbs'
+import {
+  faRectangleXmark,
+  faUser,
+  faPenToSquare,
+  faLock,
+  faHandHoldingDollar,
+  faCalendarDays,
+  faLocationDot
+} from '@fortawesome/free-solid-svg-icons'
+
+export const EventsPageDescription = () => {
+  const [event, setEvent] = useState([])
+  const [thumbsSwiper, setThumbsSwiper] = useState(null)
+  const [galeria, setGaleria] = useState([])
+  const { id } = useParams()
+  useEffect(() => {
+    cms.get(`api/events/${id}/?populate=parceires&populate=fotos_evento`).then((response) => {
+      const { data } = response.data
+      setEvent(data)
+
+      const fotoDivulgacao = data.attributes.fotos_evento
+      const images = fotoDivulgacao.data.map(
+        (image) => {
+          return {
+            id: image.id,
+            name: image.attributes?.name,
+            url: env.URL_CMS + image.attributes?.url
+          }
+        }
+      )
+      setGaleria(images)
+    })
+  }, [])
+  const handleDate = (date) => {
+    const day = date.toLocaleDateString(undefined, { day: 'numeric', Timezone: 'UTF' })
+    const month = date.toLocaleDateString('pt-BR', { month: 'long' })
+    const year = date.toLocaleDateString(undefined, { year: 'numeric', Timezone: 'UTF' })
+    return [day, month, year].join(' ')
+  }
+  const dataEmObjDate = new Date(event?.attributes?.data)
+  const indexWeek = dataEmObjDate.getDay()
+  const daysWeek = [
+    'Domingo',
+    'Segunda-Feira',
+    'Terça-Feira',
+    'Quarta-Feira',
+    'Quinta-Feira',
+    'Sexta-Feira',
+    'Sabado'
+  ]
+
+  return (
+    <div className='full-container'>
+      <EventsStyleDescription>
+        <div className='container-button-exit'>
+      <Link className='close-button' to={'/events'} ><FontAwesomeIcon icon={faRectangleXmark} size="2xl" style={{ color: '#ff4013' }} /></Link>
+        </div>
+        <div className='title'>
+          <h1>{event?.attributes?.nome}</h1>
+        </div>
+        <div className='container'>
+          <div className='text'>
+            <ul>
+              <li>
+                <p>
+                  {daysWeek[indexWeek]} - {' '}
+                  {new Date(event.attributes?.data).toLocaleDateString()}
```

Comment:
Qual a diferença desta data e da data formatada pela função `handleDate`?

Reply:
Se entendi direito, acho que toda esta formatação pode ser substituída pelo uso de `Intl.DateTimeFormat`, conforme mencionado [aqui](https://github.com/aceleradora-TW/nossa-casa-front/pull/71/files#r1344493347).

De qualquer forma, se o formato to `Intl` não servir, eu recomendo fortemente sempre passar um `locale` para a função `toLocalDateString`, senão, o comportamento desta função será diferente dependendo de onde o código estiver rodando.

Por exemplo, se nossas máquinas locais estão em português, provavelmente o node irá utilizar o locale da nossa máquina, que será `pt-BR`.

No momento que nosso código for deployado em algum servidor, e este servidor estiver nos EUA ou em qualquer outro lugar, o locale muito provavelmente não será `pt-BR`, e a data será exibida em um formato gringo feioso.